### PR TITLE
Flight panel updates

### PR DESCRIPTION
**Pull request by nsorense4**
These changes build with master.

### CODE REVIEW

File: hardware_interface/source/hyperion.c [hunk 1]
```diff
@@ -70,64 +72,56 @@ void hyperion_config_1_value(config_1_panel_t panel, config_1_channel_type_t cha
     /* Channel 1 to 3 handle temp sensor */
     case CONFIG_1_CHANNEL_TEMP_1: {
         adc_init(slave_addr, ADC_CHANNEL_0);
-        adc_get_raw(slave_addr, &data, &ch); // invalid value
-        adc_get_raw(slave_addr, &data, &ch); // correct value
+        adc_get_raw(slave_addr, &data, &ch);
```
Comment (Taylorr82) [inline hardware_interface/source/hyperion.c:75]: Did something change about how the adc works?

File: equipment_handler/source/adc_handler.c [hunk 2]
```diff
@@ -30,10 +30,11 @@
  * 		1 == success
  */
 unsigned char adc_init(uint8_t slave_addr, uint8_t channel) {
-    adc_set_command_reg(slave_addr, channel, 0, 1, 1, 0, 1);
+    adc_set_command_reg(slave_addr, channel, AD7291_EXT_REF_SET, AD7291_TSENSE_SET, AD7291_NOISE_DELAY_SET,
```
Comment (Taylorr82) [inline equipment_handler/source/adc_handler.c:33]: yay no magic numbers!

File: equipment_handler/source/adc_handler.c [hunk 3]
```diff
@@ -294,17 +295,21 @@ float adc_calculate_sensor_pd(unsigned short value, float vref) {
...
+    if (AD7291_TSENSE_AVG == true) {
+        reg_sel = AD7291_T_AVERAGE;
+    } else {
+        reg_sel = AD7291_T_SENSE;
+    }
+
     adc_set_register_pointer(slave_addr, reg_sel);
-    for (delay = 0; delay < 200000; delay++)
-        ;
+    vTaskDelay(AD7291_INIT_DELAY_TICKS);
```
Comment (Taylorr82) [inline equipment_handler/source/adc_handler.c:310]: Thanks for making this better

File: hardware_interface/source/hyperion.c (hunk 1 above)
Comment (tganley) [reply hardware_interface/source/hyperion.c:75]: It never needed to be checked twice afaik

**Review by tganley — APPROVED** [review]

File: equipment_handler/source/adc_handler.c (hunk 2 above)
Comment (nsorense4) [reply equipment_handler/source/adc_handler.c:33]: You're welcome.

**Review by yunqingyi — APPROVED** [review]

**Review by Taylorr82 — APPROVED** [review]

**Pull request merged**